modules/loader/src/test/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/metrics/MetricsSpec.scala

```diff
@@ -61,16 +62,24 @@ class MetricsSpec extends Specification {
         Some(Count(countGood, Some(countBad)))
       )
 
+      val loadResult = LoadSuccess(
+        List(
+          "contexts_com_snowplowanalytics_snowplow_test_schema_broken_1_recovered_1_1_1_737559706",
+          "contexts_com_snowplowanalytics_snowplow_test_schema_broken_1_recovered_1_0_1_1837344102"
+        )
+      )
+
       val expected = Metrics.KVMetrics.LoadingCompleted(
         KVMetric.CountGood(countGood),
         KVMetric.CountBad(countBad),
         Some(KVMetric.CollectorLatencyMin(collectorLatencyMin)),
         Some(KVMetric.CollectorLatencyMax(collectorLatencyMax)),
         KVMetric.ShredderLatencyStart(shredderStartLatency),
-        KVMetric.ShredderLatencyEnd(shredderEndLatency)
+        KVMetric.ShredderLatencyEnd(shredderEndLatency),
+        KVMetric.RecoveryTablesLoaded(2)
```

Comment:
Seems like this tells us how many recovery tables were used at all, which would include tables that existed before this load, is that correct?

I think this would be useful, but I'd also like to know whether a new recovery table was created in this batch - since I imagine support will usually be most interested in whether or not there is a new problem, when there's more than one existing recovery table in use. Is this possible to do?

(Not entirely sure I've explained the thinking correctly but shout and I'll be happy to clarify!)

Reply:
Discussed in a meeting - we provide the ability to see that information in the data we send to ops1 - it would be possible to construct a query that tells us when a new recovery table is used.

The implementation in this PR gives us the ability to get fast feedback for rollout, and tells us broadly about recovery table usage, so I'm happy that it's sufficient.